test/com/google/javascript/jscomp/CompilerTestCase.java

```diff
@@ -1793,6 +1818,45 @@ private static void transpileToEs5(AbstractCompiler compiler, Node externsRoot,
     }
   }
 
+  private static void rewriteModules(AbstractCompiler compiler, Node externsRoot, Node codeRoot) {
+    List<PassFactory> factories = new ArrayList<>();
+    CompilerOptions options = compiler.getOptions();
+    GatherModuleMetadata gatherModuleMetadata =
+        new GatherModuleMetadata(
+            compiler, options.processCommonJSModules, options.moduleResolutionMode);
+    factories.add(
+        PassFactory.builder()
+            .setName(PassNames.GATHER_MODULE_METADATA)
+            .setRunInFixedPointLoop(true)
+            .setInternalFactory((x) -> gatherModuleMetadata)
+            .setFeatureSetForChecks()
+            .build());
+    factories.add(
+        PassFactory.builder()
+            .setName(PassNames.CREATE_MODULE_MAP)
+            .setRunInFixedPointLoop(true)
+            .setInternalFactory(
+                (x) -> new ModuleMapCreator(compiler, compiler.getModuleMetadataMap()))
+            .setFeatureSetForChecks()
+            .build());
+    TranspilationPasses.addEs6ModulePass(
+        factories, new PreprocessorSymbolTable.CachedInstanceFactory());
+    factories.add(
+        PassFactory.builder()
+            .setName("REWRITE_DYNAMIC_IMPORT")
+            .setFeatureSetForChecks()
+            .setInternalFactory(
+                (x) ->
+                    new RewriteDynamicImports(
+                        compiler,
+                        compiler.getOptions().getDynamicImportAlias(),
+                        compiler.getOptions().chunkOutputType))
```

Comment:
Will we ever want this to be something other than `ChunkOutputType.ES_MODULES` here? If not, maybe we can remove that configuration option for CompilerTestCase?

Reply:
We don't need it today. I hard coded it for now and whomever needs a different option in the future can add it.